docs/conf.py

```diff
@@ -5,7 +5,7 @@
 extensions = ["sphinx.ext.autodoc", "sphinx.ext.extlinks"]
 
 exclude_patterns = ["_build"]
-source_suffix = ".rst"
+source_suffix = {".rst": "restructuredtext"}
```

Comment:
LGTM https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-source_suffix

Reply:
Yup - I saw a warning in the `poe docs` output